### PR TITLE
Add ASAM standard topic on related standards

### LINKED ISSUE #28: Add content about related standards
Add content to topics

- [ ] non_asam_standards.adoc
- [ ] other_asam_standards.adoc

### PR DESCRIPTION
**Pull request by max-rosin**
This PR resolves #28 

- [x] I have performed a self-review of my own code/documentation.
- [x] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

**Review by stefancyliax — APPROVED** [review]
This looks fine to me. Rendered document is also correct.

File: general_standard/relations_to_other_standards.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+= Relations to other standards
+
+== Positioning of Open Simulation Interface within ASAM activities
+
+OSI is part of the ASAM simulation standards that focus on simulation data for the automotive environment.
+Next to OSI, ASAM Provides other standards for the simulation domain, like OpenDRIVE, OpenSCENARIO and OpenCRG.
```
Comment (ThomasNaderBMW) [inline general_standard/relations_to_other_standards.adoc:6]: provides

File: general_standard/relations_to_other_standards.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+OpenDRIVE defines a storage format for the static description of road networks.
+In combination with OpenCRG it is possible to add very detailed road surface descriptions to the road network.
+OpenDRIVE and OpenCRG only contains static content.
```
Comment (ThomasNaderBMW) [inline general_standard/relations_to_other_standards.adoc:10]: contain

File: general_standard/relations_to_other_standards.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+To add dynamic content OpenSCENARIO is needed.
+Combined all three standards provide a scenario-driven description of traffic simulation that contains static and dynamic content.
```
Comment (ThomasNaderBMW) [inline general_standard/relations_to_other_standards.adoc:12]: Maybe this sentence can be reworked, it is a bit hard to read and understand

Comment (max-rosin) [reply general_standard/relations_to_other_standards.adoc]: This is a standard text, so I can only make minor changes.
However, I will relay your feedback.
Maybe we implement the changes in the context of #35.

**Comment by max-rosin** [comment]
Created an issue to add more content on how OSI integrates with other ASAM standards in a future release (#35 ).

**Pull request merged**